src/main/java/net/ccbluex/liquidbounce/utils/LookUtils.kt

```diff
@@ -0,0 +1,39 @@
+package net.ccbluex.liquidbounce.utils
+
+import net.minecraft.entity.Entity
+import net.minecraft.entity.EntityLivingBase
+import net.minecraft.util.Vec3
+import kotlin.math.acos
+import kotlin.math.cos
+import kotlin.math.sin
+
+object LookUtils : MinecraftInstance() {
+
+    fun isLookingOnEntities(target: Entity, lookThresholdValue: Double): Boolean {
+        val player = mc.thePlayer ?: return false
+        val playerRotation = player.rotationYawHead
+
+        if (target !is EntityLivingBase) return false
+
+        val lookVec = Vec3(
+            -sin(playerRotation * (Math.PI.toFloat() / 180f)).toDouble(),
```

Comment:
playerRotation.toRadians() can be used there =)

Reply:
like this? "val lookVec = Vec3(
            -sin(playerRotation.toRadians().toDouble()),
            0.0,
            cos(playerRotation.toRadians()).toDouble()
        ).normalize()"